src/modules/timepix3/handlers/TimingEventHandler.h

```diff
@@ -9,8 +9,11 @@
 
 #pragma once
 
+#include <atomic>
+#include <chrono>
 #include <common/kafka/EV44Serializer.h>
 #include <common/utils/EfuUtils.h>
+#include <cstdint>
 #include <dataflow/DataObserverTemplate.h>
 #include <dto/TimepixDataTypes.h>
```

Comment:
I would suggest
#include <timepix3/dataflow/DataObserverTemplate.h>
#include <timepix3/dto/TimepixDataTypes.h>

here for clarity.

But will also recommend this generally whenever including from common, efu and modules just to 
guide a future reader.